### PR TITLE
Installing dependencies for Dbench Test

### PR DESCRIPTION
**Pull request by harish-24**
As suggested, SoftwareManager is used for installing dependency

### CODE REVIEW

**Comment by apahim** [comment]
There are 2 commits in this pull request. You should probably smash them or remove the first.

**Comment by harish-24** [comment]
I have pushed as a single patch. Thanks

File: io/disk/dbench.py [hunk 1]
```diff
@@ -25,12 +26,9 @@ def setUp(self):
...
         '''
-
-        try:
-            process.run('which gcc')
-        except:
-            self.error('gcc is required by this job and is not available on'
-                       'the system')
+        sm = SoftwareManager()
+        if not sm.check_installed("gcc"):
+            sm.install("gcc")
```
Comment (apahim) [inline io/disk/dbench.py]: Will you continue the test on install failure?

Comment (apahim) [inline io/disk/dbench.py]: `install()` returns `False` on install failure. You can just:

```
...
        if not sm.check_installed("gcc") and not sm.install("gcc"):
            self.error('need gcc')
...
```

**Comment by harish-24** [comment]
@apahim  Agreed . Have sent as a new commit.

**Comment by apahim** [comment]
Looks good to me. Thank you.

**Pull request merged**